src/TestFramework/Coverage/XmlReport/XMLLineCodeCoverage.php

```diff
@@ -36,131 +36,99 @@
 namespace Infection\TestFramework\Coverage\XmlReport;
 
 use function array_key_exists;
-use function count;
-use Generator;
 use Infection\AbstractTestFramework\Coverage\CoverageLineData;
 use Infection\TestFramework\Coverage\CoverageDoesNotExistException;
 use Infection\TestFramework\Coverage\CoverageFileData;
+use Infection\TestFramework\Coverage\CoveredFileData;
 use Infection\TestFramework\Coverage\LineCodeCoverage;
 use Infection\TestFramework\Coverage\NodeLineRangeData;
-use function range;
 
 /**
+ * Used by CoveredFileData to implement LineCodeCoverage methods.
+ *
  * @internal
+ *
+ * @see CoveredFileData
```

Comment:
Do you think those `@see` annotations are really necessary? I find it a bit confusing, if I want to see the usage I can look up for the usage, at least it will be guaranteed to be up to date data whereas those `@see` are unlikely to be kept up to date

Reply:
Well, they really help be navigate the code, find what comes with what. I will certainly keep them for now, but later they are easy to remove once we're done here.